rest/types.go

```diff
@@ -0,0 +1,23 @@
+package rest
```

Comment:
Do you think it's worth naming it `restv1` upfront? FWIW The Clients API SDK encodes the v1 directly.

Reply:
Good catch. I personally don't think it's worth naming it `restv1`, but it's probably better be consistent with all the other services/APIs that we exposed. Since otherwise this just stands out like a sore thumb.